src/ObjectUtils/CMakeLists.txt

```diff
@@ -4,6 +4,18 @@
 
 cmake_minimum_required(VERSION 3.15)
 
+find_library(LLVM_LIBS LLVM NO_SYSTEM_ENVIRONMENT_PATH NO_CMAKE_SYSTEM_PATH PATHS ${LLVM_LIBRARY_DIR})
+
+if(NOT LLVM_LIBS)
+  set(LLVM_LIBS 
+  LLVMDebugInfoCodeView
+  LLVMDebugInfoDWARF
+  LLVMDebugInfoPDB
+  LLVMHeaders
+  LLVMObject
+  LLVMSymbolize)
+endif()
```

Comment:
Looking at the CI results now. I don't think this works. What exactly would you like to achieve here?

Reply:
First it tries to find libLLVM.so (this shared library includes everything and some distributions only includes this library). If not then tries to use the static libraries (LLVMDebugInfoCodeView, LLVMDebugInfoDWARF etc).

However I made two mistakes.

1. I assumed distributions either released only shared library (libLLVM.so) or static libraries. However in ubuntu's case (and probably others) they also release the shared library. This resulted `LLVM_LIBS` only using libLLVM.so

2. Because of above, LLVMHeaders, which is not a static library but a interface target which includes LLVM headers, are not included. This causes the build errors.

[aa61f65](https://github.com/google/orbit/pull/4841/commits/aa61f656aca06abd708e6a1515074b5e45333b74) should fix both issues. With this patch, it first searches the necessary llvm static libraries, if one is not found, then uses the shared library. I also readded LLVMHeaders to target_link_libraries directly.